src/pranitaPJun20/SumOfAllNumString.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Assignment 25 : 8th July 2020
+
+WAP to sum all the numbers in a given string.
+Input: J7yu9y8h1h8j4b7j3jjb6
+Ouput: Sum of all numbers in String is: 54
+ */
+package pranitaPJun20;
+
+public class SumOfAllNumString {
+	void findSumAllNum(String input) {
+		int sum = 0;
+		for (int index = 0; index < input.length(); index++) {
+			char ch = input.charAt(index);
+			if (Character.isDigit(ch)) {
+				String charToString = String.valueOf(ch);
+				int stringToInt = Integer.parseInt(charToString);
```

Comment:
Line number 16 and 17 can be replaced with below code .

Character.getNumericValue(numberString.charAt(index));